app/models/vacancy.rb

```diff
@@ -20,7 +20,7 @@ class Vacancy < ApplicationRecord
   }.merge(FLEXIBLE_WORKING_PATTERN_OPTIONS).freeze
 
   JOB_SORTING_OPTIONS = [
-    [I18n.t('jobs.sort_by.most_relevant'), ''],
+    [I18n.t('jobs.sort_by.most_relevant'), 'most_relevant'],
```

Comment:
I think this adds unnecessary complexity

Reply:
You could be right... the only reason to have 'most_relevant' is so that a user can override the default and get the 'most relevant' results for a null query, which doesn't make too much sense.